aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -8,41 +8,33 @@ By [Peter Kellner](http://peterkellner.net)
 
 The ```AnchorTagHelper``` enhances the html anchor (`<a ... ></a>`) tag. A new set of attributes are defined that work with the anchor tag. The link generated (on the href tag) is based on a combination of these new attributes that work together to form the final redirect URL. That URL can include an optional protocol such as https.
 
-For reference, the following ASP.NET ```startup.cs``` and ```SpeakerController.cs``` are defined as you would expect in a default Visual Studio .Net Core Web Project.
+For reference, the following ASP.NET ```SpeakerController.cs``` is defined as you would expect in a default Visual Studio .Net Core Web Project.
```

Comment:
Uh oh.  I removed startup.cs from the pull request I believe I made. I'm thinking @GuardRex reviewed what was already in the branch approved by @Rick-Anderson and not what was in my latest pull request.  I'm guessing I'm not following the right protocol for things to be reviewed.  Happy to update my process.  LMK.